applications/nrf_desktop/src/events/ble_event.c

```diff
@@ -23,18 +24,20 @@ static int log_ble_qos_event(const struct event_header *eh,
 {
 	const struct ble_qos_event *event = cast_ble_qos_event(eh);
 	int pos = 0;
+	char log_buf[BLE_EVENT_LOG_BUF_LEN];
 	int err;
 
-	err = snprintf(&buf[pos], buf_len - pos, "chmap:");
+	err = snprintf(&log_buf[pos], BLE_EVENT_LOG_BUF_LEN - pos, "chmap:");
 	for (size_t i = 0; (i < CHMAP_BLE_BITMASK_SIZE) && (err > 0); i++) {
 		pos += err;
-		err = snprintf(&buf[pos], buf_len - pos, " 0x%02x", event->chmap[i]);
+		err = snprintf(&log_buf[pos], BLE_EVENT_LOG_BUF_LEN - pos, " 0x%02x", event->chmap[i]);
 	}
 	if (err < 0) {
-		pos = err;
+		EVENT_MANAGER_LOG(eh, "ble_event Event Manager logging function failed");
+		return err;
```

Comment:
I would also add the ~ sign as the last one in string when log is incomplete because of too small buffer (to be consistent with previous implementation that indicated cutting log when buffer was too small)

Reply:
there is no point in adding it the way it was as `log_buf` is not used before function exit ;)

Let's remove it - we won't have to think how to implement it correctly and usefulness of it is close to none as size of the buffer is predefined. BTW the size above could be calculated depending on the actual needs but anyway.

Reply:
@zycz - github (a really bad review tool) has not added that comment into the review. Let's not forget to remove this useless ~ adding